lib/iotagent-mqtt.js

```diff
@@ -443,6 +443,7 @@ function start(newConfig, callback) {
         connectTimeout: 60 * 60 * 1000
     };
 
+    logger.format = logger.formatters.pipe;
```

Comment:
This will overwrite the format if set using the LOGOPS_FORMAT environment variable (used by https://www.npmjs.com/package/logops). It this what you really want? Maybe you could check if this environment variable is not set and set it to pipe only in this case (taking into consideration that from 1.. on, the default format for the logops package is json and not pipe as desired).